server/loops.lua

```diff
@@ -32,7 +32,7 @@ local function pay(player)
         return
     end
     local account = config.getSocietyAccount(job.name)
-    if not account or account == 0 then -- Checks if player is employed by a society
+    if not account then -- Checks if player is employed by a society
```

Comment:
Would this be better as a config option? ie. pay even if account is empty?

Reply:
I don't think we should be natively giving people the ability to use the society payment and still get paid even if the account doesn't have money. That completely negates the purpose of societal account payments.